frontend/js/src/pins/PinRecordingModal.tsx

```diff
@@ -182,6 +198,7 @@ export default NiceModal.create(
               </p>
               <div className="form-group">
                 <textarea
+                  disabled={!isCurrentUser}
```

Comment:
I believe we don't require this checker here. If I go to any user's dashboard and try pinning the listen, the condition will be false here, and we'll not be able to pin the recording.

Reply:
I think in theory a user shouldn't have an edit button to click, but I prefer to be overly cautious for future breakage, especially if it does not cost any more resources.
In this case we already calculate isCurrentUser so why not use it?
It's a "belt and braces" approach but I think it does no harm.

Reply:
We're using PinRecordingModal in two places.
1. Editing Comment
2. Pinning a Track

For the second use case, the check prevents me to pin the track if it's not my listen.

Reply:
Ah, thanks for clarifying, I hadn't caught that. Will look into it.